### PR TITLE
Update the configs for nav_env in a cleaner way

### PR DESCRIPTION
**Pull request by KavitShah1998**
The goal of this PR is to refactor construction of configs. 

Similar cleanups will be introduced inside the place_env and gaze_env refactor as well.

This would be useful inside high_level_skills

### CODE REVIEW

File: spot_rl_experiments/spot_rl/envs/nav_env.py [hunk 1]
```diff
@@ -55,6 +55,23 @@ def parse_arguments(args=sys.argv[1:]):
...
+    Constructs and updates the config for nav
+
+    Args:
+        file_path (str): Path to the config file
+        opts (list): List of options to update the config
+
+    Returns:
+        config (Config): Updated config object
+    """
+    config = construct_config(file_path=file_path, opts=opts)
```
Comment (akshararai) [inline spot_rl_experiments/spot_rl/envs/nav_env.py:69]: curious why this function is only for nav? Doesn't seem to have any nav specific properties to it? Is it the setting USE_MRCNN to false?

**Review by akshararai — APPROVED** [review]
approving, but I don't see the utility yet. Is this useful for creating configs for the tests? We discussed that we should have separate configs for the CI tests right?

File: spot_rl_experiments/spot_rl/envs/nav_env.py (hunk 1 above)
Comment (KavitShah1998) [reply spot_rl_experiments/spot_rl/envs/nav_env.py:69]: The idea for this function is to alter all the modifications needed to the config.yaml specific to navigation tasks

**Comment by KavitShah1998** [comment]
> approving, but I don't see the utility yet. Is this useful for creating configs for the tests? We discussed that we should have separate configs for the CI tests right?


The use of this is to create config for nav_env throughout the codebase anywhere needed.

**Pull request merged**